dialogs/measurement/depth_profile.py

```diff
@@ -138,10 +138,10 @@ def __init__(self, parent: BaseTab, measurement: Measurement,
         self.systematic_error = DepthProfileDialog.systerr
 
         # Checkboxes
-        self.systematic_error = DepthProfileDialog.systerr
-        self.show_scale_line = DepthProfileDialog.line_scale
-        self.show_zero_line = DepthProfileDialog.line_zero
-        self.show_used_eff = DepthProfileDialog.used_eff
+        self.systematic_error = False
+        self.show_scale_line = False
+        self.show_zero_line = False
+        self.show_used_eff = True
```

Comment:
Just changing the `used_eff` (aka `DepthProfileDialog.used_eff`) to `True` (on line 65) should be enough.

Also, `self.systematic_error` is a float, not a bool.

```suggestion
        self.systematic_error = DepthProfileDialog.systerr
        self.show_scale_line = DepthProfileDialog.line_scale
        self.show_zero_line = DepthProfileDialog.line_zero
        self.show_used_eff = DepthProfileDialog.used_eff
```

Reply:
OK